src/test/java/com/wikia/webdriver/testcases/seotests/Our404HandlerTest.java

```diff
@@ -0,0 +1,66 @@
+package com.wikia.webdriver.testcases.seotests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * @ownership Seo
+ */
+public class Our404HandlerTest extends NewTestTemplate {
```

Comment:
That's how the class is called in  code: https://github.com/Wikia/app/blob/dev/extensions/wikia/Our404Handler/SpecialOur404Handler.php